operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -77,6 +77,16 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is **Customer Education** content which can be written in first or second person to keep the audience focused and interested.
+
+For example: 
+> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
```

Comment:
```suggestion
**While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.** 
```

Reply:
Why would we not use markdown block quote here?

Reply:
I'm just keeping it aligned with the other examples in the document. But you're welcome to use Markdown block quotes if that works better. :)